src/Arcus.WebApi.Tests.Unit/Security/Authorization/JwtTokenAutorizationFilterTests.cs

```diff
@@ -11,15 +11,15 @@
 
 namespace Arcus.WebApi.Tests.Unit.Security.Authorization
 {
-    public class AzureManagedIdentityAuthorizationFilterTests
+    public class JwtTokenAutorizationFilterTests
```

Comment:
Since it's still around, we should do same tests for our old approach

Reply:
Yep, done that.